opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -178,12 +185,20 @@ def get_empty() -> "Resource":
     def attributes(self) -> Attributes:
         return self._attributes.copy()
 
+    @property
+    def schema_url(self) -> Attributes:
+        return self._schema_url
+
     def merge(self, other: "Resource") -> "Resource":
         """Merges this resource and an updating resource into a new `Resource`.
 
         If a key exists on both the old and updating resource, the value of the
         updating resource will override the old resource value.
 
+        The new `schema_url` will take an updated value only if the original
+        `schema_url` is empty. Attempting to merge two resources with
+        different, non-empty values for `schema_url` will result in an error.
```

Comment:
Maybe it'd be better to stick with the convention "old" and "updating"?
```suggestion
        The updating `schema_url` will override the old value only if the old
        `schema_url` is empty. Attempting to merge two resources with
        different, non-empty values for `schema_url` will log an error and
        set `schema_url` to the empty string.
```

Reply:
Done!